gp-includes/template.php

```diff
@@ -73,10 +73,10 @@ function gp_nav_menu_items( $location = 'main' ) {
 			$user = GP::$user->current();
 
 			$items[ gp_url( '/profile' ) ] = __('Profile');
-			$items[ gp_url( '/logout' ) ]  = __('Log out');
+			$items[ wp_logout_url( gp_url_current() ) ]  = __('Log out');
 		}
 		else {
-			$items[ gp_url_login() ] = __('Log in');
+			$items[ wp_login_url( gp_url_current() ) ] = __('Log in');
```

Comment:
Looks like none of the links are escaped currently. Maybe we should do this in `gp_nav_menu()`. (Something for a new issue/PR.)